plugin-packs/postcss-preset-env/README.md

```diff
@@ -159,6 +159,34 @@ postcss([
 - [Gulp](INSTALL.md#gulp)
 - [Grunt](INSTALL.md#grunt)
 
+### Position in your PostCSS config list
+
+[PostCSS Preset Env] only contains polyfills and fallbacks for standard CSS features.  
+[PostCSS Preset Env] is unable to handle non-standard features and syntactic sugar.
+
+If you also have PostCSS plugins for non-standard features and syntactic sugar,  
+you should place these first in the PostCSS config list.
+
+```js
+module.exports = {
+	plugins: [
+		"postcss-syntactic-sugar",
+		"postcss-non-standard",
+		// ...
+		[
+			"postcss-preset-env",
+			{
+				stage: 2,
+			},
+		],
+		// ...
+		// maybe a minifier?
+	],
+};
+```
+
+You can also use the [`insertBefore` / `insertAfter`](#insertbefore--insertafter) plugin options for more fine grained control.
```

Comment:
What's the difference between declaring plugins directly in the `plugins` array (in the correct order) and doing it through these helpers? I feel like it's the same

Reply:
With these helpers you can "insert" a plugin that isn't part of `postcss-preset-env` in between plugins bundled within `postcss-preset-env`.

With the `plugins` array you can only insert it before or after all of `postcss-preset-env`

```
- your plugin
- all of postcss-preset-env
- your plugin
```

vs.

```
- postcss-preset-env
  - bundled plugin A
  - your plugin
  - bindled plugin B
```

Reply:
Okay thanks for the clarification